plugin/package.json

```diff
@@ -70,7 +71,8 @@
   "devDependencies": {
     "@babel/core": "^7.16.0",
     "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
-    "@badeball/cypress-cucumber-preprocessor": "^12.2.0",
+    "@badeball/cypress-cucumber-preprocessor": "^19.2.0",
+    "@bahmutov/cypress-esbuild-preprocessor": "^2.2.0",
     "@cypress/browserify-preprocessor": "3.0.2",
```

Comment:
We are using esbuild-preprocessor so you can remove browserify one from package.json

Reply:
@ferhoyos I removed `@cypress/browserify-preprocessor` package